test/extended/networking/network_segmentation.go

```diff
@@ -380,6 +383,137 @@ var _ = Describe("[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:User
 				return err
 			}),
 		)
+
+		Context("UserDefinedNetwork", func() {
+			const (
+				testUdnName                = "test-net"
+				userDefinedNetworkResource = "userdefinednetwork"
+			)
+
+			BeforeEach(func() {
+				By("create tests UserDefinedNetwork")
+				cleanup, err := createManifest(f.Namespace.Name, newUserDefinedNetworkManifest(testUdnName))
```

Comment:
Since there tests are about primary network, should we use role=primary manifest, using `newPrimaryUserDefinedNetworkManifest`?

If so, `assertNetAttachDefManifest` should be changed accordingly (subents, role, etc..)

Reply:
I think I should change the name of the test, upstream this is testing secondary networks.